src/lj_jit.h

```diff
@@ -389,6 +389,9 @@ typedef struct FoldState {
   IRIns right[2];	/* Instruction referenced by right operand. */
 } FoldState;
 
+typedef void(*luaJIT_vmevent_callback)(void *data, lua_State *L,
+                                       int eventid, void *eventdata);
+
```

Comment:
conflicts with definition in `luajit.h`

Reply:
I can't really be including the luajit.h header here I will have to make a new header for it

Reply:
fixed with a new header that the eventid enum is also moved to as well as eventdata structs